crates/client/tests/app.rs

```diff
@@ -477,3 +496,71 @@ async fn dump_network_metrics() {
 
     assert_eq!(1, metrics.len());
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn agent_info() {
+    let conductor = SweetConductor::from_standard_config().await;
+    let admin_port = conductor.get_arbitrary_admin_websocket_port().unwrap();
+    let admin_ws = AdminWebsocket::connect(format!("127.0.0.1:{}", admin_port))
+        .await
+        .unwrap();
+    let _app_port = admin_ws
+        .attach_app_interface(0, AllowedOrigins::from("my_cli_app".to_string()), None)
+        .await
+        .unwrap();
+
+    let app_id: InstalledAppId = "test-app".into();
+    let agent_key = admin_ws.generate_agent_pub_key().await.unwrap();
```

Comment:
Don't need to generate an agent key I don't think? Let Holochain generate one on install and the public key will come back in the `AppInfo` from the install

Reply:
fixed in 778e6aa3335418cfbae5118b9204fe6e8a68491e